TrackingAntennaV3/main/GPS.cpp

```diff
@@ -0,0 +1,43 @@
+#include "GPS.hpp"
+#include "Logger.hpp"
+#include <Adafruit_GPS.h>
+
+//The current GPS location (latitude, longitude, and altitude)
+GPSLocation gps_location;
+
+Adafruit_GPS Adafruit_GPS(&Serial1);
+
+//Initializes the GPS module at the selected serial pins and baud rate.
+void initGPS(){
+	Adafruit_GPS.begin(9600);
+	Adafruit_GPS.sendCommand(PMTK_SET_NMEA_OUTPUT_RMCGGA);
+	Adafruit_GPS.sendCommand(PMTK_SET_NMEA_UPDATE_1HZ);
+}
+
+//Parses incoming GPS data and sends it to main
+bool parseGPS(void){
```

Comment:
is there a way we can also check if the gps was locked? Will this function return false if so?